apps/site/sentry.client.config.ts

```diff
@@ -9,13 +9,12 @@ Sentry.init({
   dsn,
   enabled: !!dsn,
   environment: process.env.NEXT_PUBLIC_VERCEL_ENV ?? "unset",
-  integrations: process.env.NEXT_PUBLIC_SENTRY_REPLAY_SESSION_SAMPLING_RATE
+  integrations: process.env.NEXT_PUBLIC_SENTRY_REPLAYS_SAMPLING_RATE
```

Comment:
When we revert this revert, we can change

`NEXT_PUBLIC_SENTRY_REPLAY_SESSION_SAMPLING_RATE`
to
`NEXT_PUBLIC_SENTRY_REPLAY_SESSION_SAMPLE_RATE`

This will better align with `sessionSampleRate`. I've already used this name in https://github.com/hashintel/hash/pull/1377